Gems/PhysX/Code/Tests/EditorMeshColliderComponentTests.cpp

```diff
@@ -147,37 +194,24 @@ namespace PhysXEditorTests
 
         EntityPtr gameEntity = CreateActiveGameEntityFromEditorEntity(editorEntity.get());
 
-        // check that the runtime entity has the expected components
-        auto* meshColliderComponent = gameEntity->FindComponent<PhysX::MeshColliderComponent>();
-        auto* staticRigidBodyComponent = gameEntity->FindComponent<PhysX::StaticRigidBodyComponent>();
-        EXPECT_TRUE(meshColliderComponent != nullptr);
-        EXPECT_TRUE(staticRigidBodyComponent != nullptr);
-
-        AzPhysics::ShapeColliderPairList shapeConfigList = meshColliderComponent->GetShapeConfigurations();
-        EXPECT_EQ(shapeConfigList.size(), 1);
-
-        for (const auto& shapeConfigPair : shapeConfigList)
+        auto checkSimulatedBody = [](const AZ::EntityId& entityId)
         {
-            EXPECT_EQ(shapeConfigPair.second->GetShapeType(), Physics::ShapeType::PhysicsAsset);
-        }
+            AzPhysics::SimulatedBody* simulatedBody = nullptr;
```

Comment:
There are a whole bunch of blocks of code which look pretty much like this one. I would be tempted to make more helper functions, so the lambda can just look something like

{
    ExpectGeometryType(entityId, physx::PxGeometryType::eSPHERE);
    ExpectAabb(entityId, min, max, tolerance);
}

Or to be even more succinct you could even do

for (auto entityId : {editorEntity->GetId(), gameEntity->GetId()})
{
    ExpectGeometryType(entityId, physx::PxGeometryType::eSPHERE);
    ExpectAabb(entityId, min, max, tolerance);
}

and not need the lambda at all.

Reply:
github mangled the indentation there :(

Reply:
Good feedback, thanks, I was looking into that myself :)